packages/kbn-coloring/src/shared_components/color_mapping/config/types.ts

```diff
@@ -134,20 +141,19 @@ export interface GradientColorMode {
 export interface Config {
   paletteId: string;
   colorMode: CategoricalColorMode | GradientColorMode;
-  assignmentMode: 'auto' | 'manual';
```

Comment:
This is the only change in the state/saved object. We don't need anymore the auto/manual mode because this are dictated by the way the other assignment is configured